### ISSUE TITLE
Don't see messages and users in chat onionshare

### ISSUE THREAD

**Issue by code1010111** [onionshare/onionshare#1391]
python = 3.9.6
tor = 0.4.6.5

![Screenshot_20210819_130133](https://user-images.githubusercontent.com/55636663/130058363-2c4af72a-ca94-423b-8f3e-335f89b75c76.png)

**Comment by twann4** [comment]
I thought I was getting the same error, but it turns out that I simply didn't wait long enough. As chats are rooted through Tor, they may be slow to appear. Did you try to wait a minute too see what happens ? Any instructions to reproduce the error ?

**Comment by code1010111** [comment]
> I thought I was getting the same error, but it turns out that I simply didn't wait long enough. As chats are rooted through Tor, they may be slow to appear. Did you try to wait a minute too see what happens ? Any instructions to reproduce the error ?

Yes, i wait 30 minutes and more. 

Instructions.... I don't know. Just git clone and start onionshare/desktop/scripts/dev.sh
no errors or warnings i don't see

**Comment by natewrench** [comment]
![image](https://user-images.githubusercontent.com/94876428/149607343-48d36861-07e0-450a-821e-33bc171bd7e6.png)
I get a blank white screen with the title and no chat

**Comment by imgustavo** [comment]
I have same issues, all other functions works, but chat is blank, i open on tor browser, brave browser, etc but msg never arrives and i dont see it on chat group also, also cant see the chat user list of people connected, using linux mint 20.3

**Comment by SaptakS** [comment]
Yes. This could be because of security settings disabling JS. @code1010111 @imgustavo can you confirm if you have JS enabled?
